minet/instagram/api_scraper.py

```diff
@@ -152,6 +158,13 @@ def request_json(self, url, magic_token=False):
         headers = {"Cookie": self.cookie}
         if magic_token:
             headers["X-IG-App-ID"] = self.magic_token
+
+        if (
+            "The link you followed may be broken, or the page may have been removed."
+            in requests.get(url, headers=headers).text
```

Comment:
Why are you using requests for this? You are making the same call below using `request_json`, which means you are actually making 2 http calls per url. Maybe a good solution would be to use `request` (the minet function, not the external library `requests` which is not a dependency of minet), to read its decoded text, then parse it instead. Check how `request_json` is written for instance.